README.md

```diff
@@ -18,8 +18,9 @@ For more information, see the blog post [Nextcloud in Digital Imaging](https://n
 
 ### Features
 
-* **Viewer:** A DICOM viewer which displays images grouped by study and series in selected folders and allows to manipulate images with imaging tools
-* **DICOM Dump:** A list of DICOM attributes displayed on the sidebar with image thumbnail
+* **Viewer:** A DICOM viewer which displays images grouped by study and series in selected folders and allows to manipulate images with imaging tools.
+* **DICOM Dump:** A list of DICOM attributes displayed on the sidebar with image thumbnail (*Note:* The Nextcloud files app plugin "Comments" must be activated to use this).
```

Comment:
Please remove the note which requires comments app. It is a bug and i created the issue #45 to fix it.

Reply:
done